.github/workflows/sdks.yml

```diff
@@ -480,3 +484,85 @@ jobs:
+          fail-on-cache-miss: true
+      - name: Get cached SDK (armv7)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-armv7-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-armv7-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Assemble SDK
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        id: assemble-sdk
+        run: |
+          set -x
+          ls -lah ~/swift-*-sdk.tar.xz
+
+          # e.g. swift-5.10.1-RELEASE-android-sdk
+          SDK_DIR=${{ steps.version.outputs.tag }}-android-sdk
+          echo "Merging $(echo $(ls -1)) into SDK_DIR: ${SDK_DIR}"
+
+          mkdir ${SDK_DIR}
+
+          for SDK_PATH in ~/swift-*-sdk.tar.xz; do
+            MERGEDIR=$(basename $SDK_PATH .tar.xz)
+            mkdir ${MERGEDIR}
+            pushd ${MERGEDIR}
+            # e.g. swift-android-sdk-release-aarch64/swift-release-android-aarch64-24-sdk.tar.xz
+            tar xf ${SDK_PATH}
+            tree swift-*-sdk
+            pushd swift-*-sdk
+
+            ARCH=$(ls usr/lib/swift_static/android/*/swiftrt.o | cut -f 5 -d '/')
+
+            TRIPLE="${ARCH}-linux-android"
+            if [[ $ARCH == 'armv7' ]]; then
+                TRIPLE="arm-linux-androideabi"
+            fi
+
+            echo "Creating ${TRIPLE} from ${MERGEDIR}…"
+
+            # remove the destination files, which include invalid absolute paths
+            rm usr/swiftpm-*.json
+
+            mkdir -p usr/lib/${TRIPLE}
+            rm -r usr/bin
+            mv usr/lib/swift_static/android/*.a usr/lib/swift_static/android/${ARCH}/
+            mv usr/lib/*.so* usr/lib/swift/android/*.so* usr/lib/${TRIPLE}
+            mv usr/lib/pkgconfig usr/lib/${TRIPLE}
+            rm -r usr/lib/swift/pm
+            # clear the symlink to /home/runner/work/swift-android-sdk/swift-android-sdk/sdk-config/swift-5.10.1-RELEASE-ubuntu22.04/usr/lib/clang/15.0.0
+            # we will reference it with an -I flag in the destination.json instead
+            rm usr/lib/swift_static/clang
+            rm usr/lib/swift/clang
+
+            popd
+            popd
+
+            # copy all the processed SDK files into the SDK_DIR
+            rsync -ar ${MERGEDIR}/swift-*-sdk/ "${SDK_DIR}"
+          done
+
+          # output visualization of the SDK_DIR file tree
+          tree "${SDK_DIR}"
+
+          echo "Merged toolchains into ${SDK_DIR}"
+          du -skh "${SDK_DIR}"
+          tar cJf "${SDK_DIR}".tar.xz "${SDK_DIR}"
+          du -skh "${SDK_DIR}".tar.xz
+          shasum -a 256 "${SDK_DIR}".tar.xz
+      - name: Upload Assembled SDK
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ steps.version.outputs.tag }}-android-sdk.tar.xz
```

Comment:
Write `$SDK_DIR` to `$GITHUB_OUTPUT` in the previous step then you can simply copy it here with the API level included.